pod/perlfunc.pod

```diff
@@ -6561,32 +6561,60 @@ X<rand> X<random>
 
 =for Pod::Functions retrieve the next pseudorandom number
 
-Returns a random fractional number greater than or equal to C<0> and less
+Returns a random fractional number greater than or equal to C<0> and I<less>
```

Comment:
Somehow I think we use B<> instead of italics for these situations

Reply:
Fair enough. I landed this change.